### PR TITLE
LAW-292: chartbeat tracking

### PR DESCRIPTION
**Pull request by dashaluna**
**NOTE**: depends on #68

### CODE REVIEW

**Comment by mattheu** [comment]
Same comment as the #68 - I think this is a little overly complex. There are already simple chartbeat script template functions, just call them directly in the template.

File: simple-facebook-instant-articles.php [hunk 1]
```diff
@@ -736,6 +737,26 @@ protected function get_simple_reach_analytics_code() {
...
+	protected function get_chartbeat_analytics_code() {
+
+		if (
+			lawrence_option_enabled( 'usat_chartbeat_enabled' )
+			&& function_exists( 'usat_chartbeat_add_header' )
+			&& function_exists( 'usat_chartbeat_add_footer' )
+		) {
+			ob_start();
+			require( trailingslashit( $this->template_path ) . 'script-chartbeat.php' );
+			return ob_get_clean();
```
Comment (dashaluna) [inline simple-facebook-instant-articles.php:755]: @mattheu I've noticed this pattern is repeated a lot. Do you think it's worth making a method for it or it's not too bad to repeat the 3 lines?

Comment (mattheu) [inline simple-facebook-instant-articles.php:755]: actually yes I think it would be nice to have a render_template method that we can reuse

Comment (dashaluna) [inline simple-facebook-instant-articles.php:755]: Let's have it as a separate PR? I think this one is ok to merge if you're happy with it

**Pull request merged**